templates/contact-us.html

```diff
@@ -0,0 +1,136 @@
+{% extends "base.html" %}
+
+{% block meta_title %}
+  Contact-us
+{% endblock meta_title %}
+
+{% block meta_description %}
+  Contact-us form for reaching out design team at Canonical
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  <!-- to be added -->
+{% endblock meta_copydoc %}
+
+{% block content %}
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h1>Be part of the open design community</h1>
+      </div>
+      <div class="col">
+        <p>
+          The Open Design Working Group (ODWG) at Canonical is dedicated to empowering external designers to contribute to open-source projects. We focus on bridging the gap between designers and project maintainers, making it easier for designers to get involved and for maintainers to receive valuable design input.
+        </p>
+        <p>
+          Join us in shaping the future of design within the open-source community &mdash; and don't forget to share the sauce!
+        </p>
+        <div class="p-notification--information">
+          <div class="p-notification__content">
+            <h5 class="p-notification__title">Call for feedback</h5>
+            <p class="p-notification__message">
+              If you'd like a simple start try sending feedback to our <a href="https://docs.google.com/document/d/11kQqpy4WqvqYDe9VS547uoTI4CfXriaBhTQMhYR5uo0/edit?usp=sharing">design contribution document</a>.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="p-image-container--cinematic">
+        {{ image(url="https://assets.ubuntu.com/v1/4206dcc7-sauce_hero.svg",
+                alt="",
+                width="2400",
+                height="1000",
+                hi_def=True,
+                attrs={"class": "p-image-container__image"},
+                loading="auto|lazy") | safe
+        }}
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Connect with us</h2>
+      </div>
+      <div class="col">
+        <form action="https://ubuntu.com/marketo/submit"
+              method="post"
+              class="p-form p-form--stacked">
+          <div class="p-section--shallow">
+            <label for="firstName" class="is-required">First name:</label>
+            <input required id="firstName" name="firstName" maxlength="255" type="text" />
+            <label for="lastName" class="is-required">Last name:</label>
+            <input required id="lastName" name="lastName" maxlength="255" type="text" />
+            <label for="email" class="is-required">Email:</label>
+            <input required id="email" name="email" maxlength="255" type="text" />
+            <label for="comments" class="is-required">Anything else you'd like to share?</label>
+            <textarea required id="comments" name="comments" rows="5" maxlength="2000"></textarea>
```

Comment:
Have you checked the name of this field in marketo, usually it is something like commentForLead